ArcGISRuntimeSDKQt_QMLSamples/Geometry/ConvexHull/ConvexHull.qml

```diff
@@ -0,0 +1,127 @@
+// [WriteFile Name=ConvexHull, Category=Geometry]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.8
+import QtQuick.Layouts 1.11
+import QtQuick.Controls 2.6
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapTopographic {}
+        }
+
+        GraphicsOverlay {
+            id: graphicsOverlay
+
+            Graphic {
+                id: inputsGraphic
+                SimpleMarkerSymbol {
+                    id: markerSymbol
+                    style: Enums.SimpleMarkerSymbolStyleCircle
+                    size: 10
+                    color: "red"
+                }
+            }
+
+            Graphic {
+                id: convexHullGraphic
+            }
+        }
+
+        SimpleFillSymbol {
+            id: fillSymbol
+            style: Enums.SimpleFillSymbolStyleNull
+            color: "transparent"
+            SimpleLineSymbol {
+                id: lineSymbol
+                style: Enums.SimpleLineSymbolStyleSolid
+                width: 3
+                color: "blue"
+            }
+        }
+
+        // stores clicked points
+        PointCollection {
+            id: inputs
+            spatialReference: mapView.spatialReference
+        }
+
+        MultipointBuilder {
+            id: multipointBuilder
+            spatialReference: mapView.spatialReference
+        }
+
+        onMouseClicked: {
+            var clickedPoint = mapView.screenToLocation(mouse.x, mouse.y);
+            inputsGraphic.geometry = clickedPoint;
+            inputs.addPoint(clickedPoint);
+            multipointBuilder.points = inputs;
```

Comment:
I believe you can just do: `multipointBuilder.points.addPoint(clickedPoint)`.  
Remove the `PointCollection` you have created.